sql/postgresql/9.0-GA.sql

```diff
@@ -0,0 +1,42 @@
+-- *********************************************************************
+-- Update Database Script
+-- *********************************************************************
+-- Change Log: META-INF/resourcesPrivate/com.symphonicsoft.db.cli/liquibase/symphonic-changelog.xml
+-- Liquibase version: 3.10.2
+-- *********************************************************************
+
+SET SEARCH_PATH TO public;
+
+-- Lock Database
+UPDATE public.databasechangeloglock SET LOCKED = TRUE, LOCKEDBY = '127.0.0.1 (127.0.0.1)', LOCKGRANTED = '2021-12-13 11:08:53.096' WHERE ID = 1 AND LOCKED = FALSE;
+
+SET SEARCH_PATH TO public;
+
+SET SEARCH_PATH TO public;
+
+-- Changeset META-INF/resourcesPrivate/com.symphonicsoft.db.cli/liquibase/9.0-GA/changelog.xml::1::symphonic
+SET SEARCH_PATH TO public;
```

Comment:
I'm guessing this was automatically generated but do you know why the same line is duplicated 3 times?

Reply:
Yes, it is auto generated. I don't know the source of the duplication, but as long as it's idempotent it should be ok.